rtil/src/native/font.rs

```diff
@@ -0,0 +1,43 @@
+use std::cell::Cell;
+use crate::native::reflection::{ObjectWrapper, UObject};
+use crate::native::uworld::ENGINE_INDEX;
+use crate::native::{ArrayWrapper, StructValueWrapper, UeScope, UFONTBULKDATA_INITIALIZE};
+use std::sync::atomic::Ordering;
+use std::mem;
+
+pub struct UFont(*mut UFontUE);
+pub enum UFontUE {}
+
+// WARNING: somewhat unsound - see AMyCharacter
+unsafe impl Send for UFont {}
+
+#[repr(C)]
+struct UFontBulkData {
+    // stub, we only need a pointer to this struct
+}
+
+impl UFont {
+    pub fn get_font(name: &str) -> *mut UFontUE {
+        UeScope::with(|scope| {
+            let engine = scope.get(ENGINE_INDEX.get().unwrap());
+            let font = engine.get_field(name).unwrap::<ObjectWrapper>();
+            font.as_ptr() as *mut UFontUE
+        })
+    }
+
+    pub fn set_font(font: *mut UFontUE) {
```

Comment:
Unsound as I can pass in `std::ptr::null_mut()` from safe code and make the function crash.

If you only want to allow items gotten from `get_font`, you can create a wrapper-struct with the pointer as a private field. Instances of that struct can only be created by using `get_font` and thus the correctness of the pointer is guaranteed.